specification/compute/resource-manager/Microsoft.Compute/stable/2021-07-01/compute.json

```diff
@@ -3984,33 +4600,45 @@
             "description": "The name of the resource group."
           },
           {
-            "name": "restorePointCollectionName",
+            "name": "capacityReservationGroupName",
             "in": "path",
             "required": true,
             "type": "string",
-            "description": "The name of the restore point collection."
+            "description": "The name of the capacity reservation group."
           },
           {
-            "name": "restorePointName",
+            "name": "capacityReservationName",
             "in": "path",
             "required": true,
             "type": "string",
-            "description": "The name of the restore point."
+            "description": "The name of the capacity reservation."
+          },
+          {
+            "name": "$expand",
+            "in": "query",
+            "required": false,
+            "type": "string",
+            "description": "The expand expression to apply on the operation. 'InstanceView' will retrieve the list of instance views of the capacity reservation.",
```

Comment:
The expand expression to apply on the operation. 'InstanceView' retrieves a snapshot of the runtime properties of the capacity reservation that is managed by the platform and can change outside of control plane operations.

Reply:
updated